src/lib/is-installed.ts

```diff
@@ -2,39 +2,139 @@ import { app } from 'electron';
 import fs from 'fs';
 import path from 'path';
 
-let appPaths: Record< keyof InstalledApps, string[] >;
+type InstalledApp =
+	| 'vscode'
+	| 'phpstorm'
+	| 'cursor'
+	| 'windsurf'
+	| 'nova'
+	| 'webstorm'
+	| 'sublime'
+	| 'atom';
```

Comment:
Couple of notes regarding this list:
- atom was archived in 2022
- nova is mac-only